libraries/AP_BattMonitor/AP_BattMonitor_SMBus.h

```diff
@@ -93,11 +126,17 @@ class AP_BattMonitor_SMBus : public AP_BattMonitor_Backend
     bool _pec_supported; // true if PEC is supported
 
     int32_t _serial_number = -1;    // battery serial number
-    uint16_t _full_charge_capacity; // full charge capacity, used to stash the value before setting the parameter
+    int32_t _full_charge_capacity; // full charge capacity, used to stash the value before setting the parameter
```

Comment:
Why change this? It's a word off the SMBus, so it's only 16 bits. Increasing it here just uses more memory.

Reply:
Thanks! I had the frontend and backend capacity getters initially to uint16 but realized with only 65,536 mAh that they should likely be int32 but took that change too far to the SMBus.